dataactvalidator/filestreaming/fieldCleaner.py

```diff
@@ -126,31 +131,26 @@ def cleanRow(cls, row, fileType, validationInterface):
                     # Replace empty strings with null
                     value = None
 
-                row[key] = cls.padField(key,value,fileType,validationInterface)
-
+                row[key] = cls.padField(field,value)
         return row
 
     @staticmethod
-    def padField(field,value,fileType,validationInterface):
+    def padField(field, value):
         """ Pad value with appropriate number of leading zeros if needed
 
         Args:
-            field: Name of field
+            field: FileColumn object
             value: Value present in row
-            fileType: Type of file data is being loaded for
-            interfaces: InterfaceHolder object
 
         Returns:
             Padded value
         """
         # Check padded flag for this field and file
-        if value is not None and validationInterface.isPadded(field,fileType):
-            # If padded flag is true, get column length
-            padLength = validationInterface.getColumnLength(field, fileType)
+        if value is not None and field.padded_flag and field.length is not None:
```

Comment:
This answers my previous question about handling nulls for the `length` field in the database. It looks like we're handling the `NULL` values instead of just handling a default value.